src/data-table/data-table-column.js

```diff
@@ -19,6 +21,8 @@ const defaultProps = {
   format: null,
   template: null,
   nullTemplate: null,
+  sortable: false,
+  sortType: 'basic',
```

Comment:
maybe set default `sortType` to string as we mostly use default string sort in snoke and overlord